scripts/publishToConfluence.gradle

```diff
@@ -71,6 +71,11 @@ task publishToConfluence(
             findProperty("confluence.useV1Api") : config.confluence.useV1Api != [:] ?
             config.confluence.useV1Api :true
         Asciidoc2ConfluenceTask.From(config, docDir).execute()
+        if (config.confluence.pages.isEmpty()) {
```

Comment:
i don't hink this will work properly since there is no such property `pages`. `config` is the object created from `docToolchainCongig.groovy` 
Above we explicitly overwrite some values if they are provided via Gradle CLI params like `-PuseV1Api=true`

Reply:
to be honest, I haven't checked the code yet. It was generated by copilot workspaces and I hoped to find a good idea in here...